BC-Gov-Org-HowTo/Cheatsheet.md

```diff
@@ -0,0 +1,76 @@
+
+
+### How to License the Contents of a Repository
+- Ensure [authority to LICENSE](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Licenses.md) the code and other non-code content in the repository
+- Choose [a LICENSE](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Licenses.md) for your code (.md files are considered code). The default is the Apache 2.0 LICENSE
+- 1-2-3 Apply the Apache 2.0 LICENSE:
+  1. Attach appropriate LICENSE file directly in the repository
+  2. Add the boiler-plate Apache 2.0 LICENSE to the the bottom of your README.md
+  3. Add the boiler-plate Apache 2.0 LICENSE to the comments header of *every source code file* 
+- Choose a license for any other content (e.g. docs, wikis and non-code stuff) --- default is [Creative Commons Attribution 4.0 International](Creative Commons Attribution 4.0 International, and should look like this at the bottom of your README.md) --- and add to the footer of your README.md
+- Repositories can have multiple licenses, for example if there is a mix of code and non-code or if the repo contains open datasets under more than one license
+
+### Privacy, Security & Intellectual Property/Copyright
+-  Completing the [Open Content Assessment Checklist](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Content-Approval-Checklist.md) helps determine if content can be posted to GitHub by ensuring:
+  1. The content is free of [Personal Information](http://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy) (Privacy)
+  2. The content has been labelled 'Public' using the  [Information Security Classification Framework](http://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification-framework) (Security)
+  3. The content is fully owned by the B.C. government and/or the B.C. government holds the [Intellectual Property/Copyright](http://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/intellectual-property) for the content
+  4. There are no other legal, contractual or policy constraints
+- Read more about privacy, security and intellectual property in the [BC-Policy-Framework-For-GitHub](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/COI-Priv-IP.md)
+
+
+### Approval Requirements
+-  Approvals and minium requirements are set out in the
+-  [Open Development Standard (pages 14-15)](http://www2.gov.bc.ca/assets/gov/government/services-for-government-and-broader-public-sector/information-technology-services/standards-files/development_standards_for_information_systems_and_services.pdf)
```

Comment:
It is [here](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/ref-docs/96184_Open_Source_Guideline.pdf) and referenced in the Employee Guide README.md, the Principles.md (3rd bullet), and first sentence of Content-Approval-Checklist.md -- although not linked to the PDF.